crates/forge/src/main.rs

```diff
@@ -224,6 +226,26 @@ fn main_execution() -> Result<bool> {
     Ok(all_failed_tests.is_empty())
 }
 
+#[allow(clippy::too_many_lines)]
+fn main_execution() -> Result<bool> {
+    let cli = Cli::parse();
+
+    which::which("scarb")
+        .context("Cannot find `scarb` binary in PATH. Make sure you have Scarb installed https://github.com/software-mansion/scarb")?;
+
+    match cli.subcommand {
+        Subcommands::Init { name } => {
```

Comment:
```suggestion
        Subcommand::Init { name } => {
```
please use singular form for enums

Reply:
It will conflict with `clap::Subcommand`, but can change it to `ForgeSubcommand`